acceptance/framework/vault/vault_cluster.go

```diff
@@ -149,6 +149,25 @@ func (v *VaultCluster) bootstrap(t *testing.T, vaultNamespace string) {
 
 	namespace := v.helmOptions.KubectlOptions.Namespace
 	vaultServerServiceAccountName := fmt.Sprintf("%s-vault", v.releaseName)
+
+	// In Kube 1.24+ the serviceAccount does not have a secret automatically generated so we will create one prior to
+	// needing to use it in ConfigureAuthMethod. Vault expects the user to create the secret manually for any
+	// Kube AuthMethod use.
+	retry.Run(t, func(r *retry.R) {
+		sa, err := v.kubernetesClient.CoreV1().ServiceAccounts(namespace).Get(context.Background(), vaultServerServiceAccountName, metav1.GetOptions{})
+		require.NoError(r, err)
+		if len(sa.Secrets) == 0 {
+			_, err = v.kubernetesClient.CoreV1().Secrets(namespace).Create(context.Background(), &corev1.Secret{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:        vaultServerServiceAccountName,
+					Annotations: map[string]string{"kubernetes.io/service-account.name": vaultServerServiceAccountName},
+				},
+				Type: "kubernetes.io/service-account-token",
```

Comment:
There is a constant for the Type that we could use across all the secrets to make them look nicer!!
```suggestion
				Type: corev1.SecretTypeServiceAccountToken,
```